client/packages/system/src/Stock/api/api.ts

```diff
@@ -70,13 +70,17 @@ export const getStockQueries = (stockApi: StockApi, storeId: string) => ({
       nodes: StockLineRowFragment[];
       totalCount: number;
     }> => {
+      const filter = {
```

Comment:
Andrei said to filter in FE... but Stocktake by Masterlist and Location are generated in BE so 🤷🏻